src/_includes/get_in_touch.html

```diff
@@ -0,0 +1,51 @@
+<section class="get-in-touch">
+  {% include page_location_anchor.html anchor_id="get-in-touch" %}
+
+  <div class="get-in-touch__form">
+    {% include section_intro.html description=include.description title=include.title %}
+
+    <div class="sm-enquiry">
```

Comment:
Not a blocker but it would be nice to bring this class name in line without our BEM style block,
e.g. `get-in-touch__form-inner` or something better